File: gateOne/day-One/python/taskten.py
sum = 0
score_number = 10
index = 1
for count in range(score_number):
	score_collector = int(input("enter number: "))
	if score_collector >= 0 and score_collector <= 100:
		sum += score_collector
		index += 1

average = sum / index
print("the average of is ", average)

File: gateOne/day-One/java/TaskTen.java
import java.util.Scanner;
public class TaskTen{
	public static void main(String... args){

	Scanner input = new Scanner(System.in);
	int sum = 0;
	int scoreNumber = 10;
	int index = 1;
	for(int counter = 1; counter <= scoreNumber; counter++){
		int scoreCollector = input.nextInt();
		if(scoreCollector >= 0 && scoreCollector <= 100){
			sum += scoreCollector;
			index++;
		}
	}

	double average = sum / (double)index
	System.out.print("the average of is " + average);

	}




}